sourcegraph/sourcegraph
# Simplify telemetry structure; eliminate difference between critical and non-critical telemetry

Proof of concept PR here. I'm failing in Buildkite and am struggling resolving some of them (I can't even seem to get prettier to run anymore, it's been too long). I would love to work with @andersonlauren and @ryphil to (1) make specific reductions to our telemetry as part of this or soon after, and (2) get eng support to fully clean up and remove the old config option.


## Test plan

* Run a local instance, set disableNonCriticalTelemetry to true
* See if any errors appear
* Confirm telemetry sent is identical regardless of the value of the configuration option above
